Connectors/Revit/Speckle.Connectors.RevitShared/Operations/Receive/RevitHostObjectBuilder.cs

```diff
@@ -47,7 +48,12 @@ CancellationToken cancellationToken
   ) =>
     _syncToThread.RunOnThread(() =>
     {
-      var objectsToConvert = _traverseFunction.Traverse(rootObject).Where(obj => obj.Current is not Collection);
+      using var activity = SpeckleActivityFactory.Start("Build");
+      IEnumerable<TraversalContext> objectsToConvert;
+      using (var _ = SpeckleActivityFactory.Start("Traverse"))
```

Comment:
Since traversal is just returning a lazily enumerated `IEnumerable` here, I can't see this activity being useful...